src/components/Calculator.js

```diff
@@ -1,30 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Screen from './Screen';
 import Button from './Button';
+import calculate from '../logic/Calculator';
 
 function Calculator() {
+  const [calculation, setCalculation] = useState({
+    total: null,
+    next: null,
+    operation: null,
+  });
+
+  const makeCalculations = (e) => {
+    const buttonName = e.currentTarget.textContent;
+    const newState = calculate(calculation, buttonName);
+    setCalculation(newState);
+  };
+
+  const { total, next, operation } = calculation;
+
   return (
     <div className="w-[9cm] md:w-[12cm] h-[9cm] md:h-[12cm] border m-10 grid grid-cols-4 grid-rows-6">
-      <Screen />
-      <Button value="AC" />
-      <Button value="+/-" />
-      <Button value="%" />
-      <Button value="÷" classextend="bg-orange-500" />
-      <Button value="7" />
-      <Button value="8" />
-      <Button value="9" />
-      <Button value="×" classextend="bg-orange-500" />
-      <Button value="4" />
-      <Button value="5" />
-      <Button value="6" />
-      <Button value="-" classextend="bg-orange-500" />
-      <Button value="1" />
-      <Button value="2" />
-      <Button value="3" />
-      <Button value="+" classextend="bg-orange-500" />
-      <Button value="0" classextend="col-span-2 bg-slate-200 text-black" />
-      <Button value="." />
-      <Button value="=" classextend="bg-orange-500" />
+      <Screen total={total} next={next} operation={operation} />
+      <Button handleClick={(e) => makeCalculations(e)} value="AC" />
+      <Button handleClick={(e) => makeCalculations(e)} value="+/-" />
+      <Button handleClick={(e) => makeCalculations(e)} value="%" />
+      <Button handleClick={(e) => makeCalculations(e)} value="÷" classextend="bg-orange-500" />
+      <Button handleClick={(e) => makeCalculations(e)} value="7" />
+      <Button handleClick={(e) => makeCalculations(e)} value="8" />
+      <Button handleClick={(e) => makeCalculations(e)} value="9" />
+      <Button handleClick={(e) => makeCalculations(e)} value="×" classextend="bg-orange-500" />
```

Comment:
- [ ] I got this error in the process of multiplying two numbers, kindly fix it. 👍 Perhaps you need to replace it with the correct symbol, which is the standard multiplication operator in JavaScript. 👍 

<img width="545" alt="Untitled" src="https://github.com/AsumVictor/Math-magician/assets/103386183/c5da948d-a957-4b6e-9101-f74b0da85b8d">

Reply:
Thank you, I have solved it.